ui/src/app/cohort-search/node-info/node-info.component.ts

```diff
@@ -73,7 +73,7 @@ export class NodeInfoComponent implements OnInit, OnDestroy, AfterViewInit {
 
   get paramId() {
     return `param${this.node.conceptId && !this.isPPI ?
```

Comment:
nit: can we spell out PPI instead of abbreviating it?

Reply:
Good idea. Actually, the way we handle domains has changed, so I'm renaming to `isSurvey` to better reflect what we're checking here.